jdaviz/configs/imviz/plugins/catalogs/catalogs.py

```diff
@@ -62,7 +67,10 @@ class Catalogs(PluginTemplateMixin, ViewerSelectMixin, HasFileImportSelect, Tabl
     def user_api(self):
         return PluginUserApi(self, expose=('clear_table', 'export_table', 'import_catalog',
                                            'zoom_to_selected', 'select_rows',
-                                           'select_all', 'select_none'))
+                                           'select_all', 'select_none',
+                                           'results_available', 'number_of_results',
```

Comment:
is it useful for the user to have access to `results_available` and `number_of_results` or is that just used internally?

Otherwise code changes look good - thanks for updating the tests too!

Reply:
probably not, ill remove those